pkg/cmd/server/api/v1/types.go

```diff
@@ -970,8 +970,16 @@ type KubernetesMasterConfig struct {
 	ServicesNodePortRange string `json:"servicesNodePortRange"`
 	// StaticNodeNames is the list of nodes that are statically known
 	StaticNodeNames []string `json:"staticNodeNames"`
+
 	// SchedulerConfigFile points to a file that describes how to set up the scheduler. If empty, you get the default scheduling rules.
 	SchedulerConfigFile string `json:"schedulerConfigFile"`
+	// SchedulerName is the name this scheduler uses to assign pods.
```

Comment:
We *can* group them - except we'd have to move scheduler config (to be
consistent with upstream).  Upstream has two configs - the policy file, and
then the "component config" that wraps it.  I don't like having scheduler
config file at the root and the other scheduler config in a sub object, but
I could be convinced.  I'd argue we'd fix this by splitting it out
eventually.

On Wed, Jul 27, 2016 at 3:52 PM, Jordan Liggitt <notifications@github.com>
wrote:

> In pkg/cmd/server/api/v1/types.go
> <https://github.com/openshift/origin/pull/10064#discussion_r72510463>:
>
> >  	// SchedulerConfigFile points to a file that describes how to set up the scheduler. If empty, you get the default scheduling rules.
> >  	SchedulerConfigFile string `json:"schedulerConfigFile"`
> > +	// SchedulerName is the name this scheduler uses to assign pods.
>
> I'm sort of surprised these options weren't already part of the scheduler
> config object read from the file
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin/pull/10064/files/b4296d1dbcfd5294ac9467d4d42ad735d1cfbc5d#r72510463>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABG_p4Lwhdvxg6K-Glh-pzVW032YvinVks5qZ7bwgaJpZM4JWjvW>
> .
>